oteljava/metrics/src/main/scala/org/typelevel/otel4s/oteljava/metrics/CounterImpl.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.otel4s
+package oteljava
+package metrics
+
+import cats.effect.kernel.Sync
+import io.opentelemetry.api.metrics.{LongCounter => JLongCounter}
+import org.typelevel.otel4s.meta.InstrumentMeta
+import org.typelevel.otel4s.metrics._
+import org.typelevel.otel4s.oteljava.AttributeConverters._
+
+import scala.collection.immutable
+
+private[oteljava] class CounterImpl[F[_]](
```

Comment:
This class was removed in https://github.com/typelevel/otel4s/pull/449.

Reply:
oops. did I do a bad rebase? D: